packages/maskbook/src/utils/type-transform/BackupFormat/JSON/latest.ts

```diff
@@ -28,3 +38,15 @@ export function UpgradeBackupJSONFile(json: object, identity?: ProfileIdentifier
     }
     return null
 }
+
+export function getBackupPreviewInfo(json: BackupJSONFileLatest): BackupPreview {
+    return {
+        email: 'alice@example.com', // TODO: email
```

Comment:
Omit this field since it is not available yet.

Reply:
That's why I add an example email here for UI preview. Will change this when the account system is ready.